Cesium.IntegrationTests/Cesium.IntegrationTests.proj

```diff
@@ -1,4 +1,9 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="Current">
+
+    <PropertyGroup>
+        <NoWarn>NU1503</NoWarn>
```

Comment:
This one warning means that we have incomplete project file, and we should provide required target, even if would be empty one like other targets below.

Reply:
I'd vote for adding a target framework (`net6.0`), even if unused. With a comment explaining the issue, of course.

Reply:
I would add `/bl` switch, look at the file and the provide necessary target. I think having just empty `_IsProjectRestoreSupported` target would be fine.